site-src/concepts/versioning.md

```diff
@@ -2,59 +2,102 @@
-Gateway API provides 2 release channels: an Experimental one and a Standard one.
+* Validating Webhook (Will be deprecated in v1.0 release)
+
+### Release Channels
+Release channels are used to indicate feature stability within Gateway API. All
+new features and resources start in the Experimental release channel. From that
+point, these may graduate to the Standard release channel or be dropped from the
+API entirely.
+
+The following diagram provides a high level overview of the lifecycle of a
+feature or resource proposed by a new [GEP](/geps/overview/) (enhancement
+proposal) in Gateway API:
+
+```mermaid
+flowchart TD
+    0([Implementable GEP]) --> A
+    A>Experimental Channel] --> B([Widely used and working well?])
+    B -->|Yes| C>Standard Channel]
+    B -->|No| D([Could Changes Help?])
+    D -->|Yes| E([Adjust and try again])
+    D -->|No| F>Remove From API]
+    E -->A
+
+style A fill:#eeb
+style C fill:#beb
+style F fill:#ebb
+```
 
 The Standard release channel includes:
 
-* Resources that have graduated to beta
-* All fields that have graduated to standard and are no longer considered
-  experimental
+* Resources that have graduated to Beta or GA API Versions (note that Beta API
+  versions are being phased out in Gateway API)
+* All fields that have graduated to Standard from the Experimental Channel
 
 The Experimental release channel includes everything in the Standard release
 channel, plus:
 
-* Alpha API resources
-* New fields that are considered experimental and have not yet graduated to the
-  standard channel
+* Resources with Alpha API Versions
+* All new fields before they graduate to Standard Channel
 
 ![Release Channel Overlap](/images/release-channel-overlap.svg)
 <!-- Source: https://docs.google.com/presentation/d/1sfZTV-vlisDUIie_iK_B2HqKia_querT6m6T2_vbAk0/edit -->
 
-Release channels are used to indicate feature stability within Gateway API. For
-example, even if a resource has already graduated to beta, we need a way to test
-out new fields and features before committing to their long term stability. With
-upstream Kubernetes APIs, this is accomplished with [feature
-gates](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/).
-Unfortunately, CRDs do not have a similar concept yet.
-
-Instead of trying to recreate feature gates in this project, we've introduced
-release channels. Starting in v0.5.0, all new fields and features will start in
-the Experimental release channel. From that point, these may graduate to the
-Standard release channel or be dropped from the API entirely. See
-[GEP-922](/geps/gep-922) for a more detailed discussion of this approach to new
-fields and features.
-
-![Lifecycle of New Gateway API Fields](/images/lifecycle-new-fields.svg)
-<!-- Source: https://docs.google.com/presentation/d/1sfZTV-vlisDUIie_iK_B2HqKia_querT6m6T2_vbAk0/edit -->
+We recommend using the "Standard" release channel by default as it will provide
```

Comment:
Should we mention this is a recommendation for users, not for implementors? I'm hoping to open my implementor's guide PR later today which will suggest that implementors use experimental structs for development, since those will work with either CRD.

Reply:
Updated, thanks!